### ISSUE TITLE
Improve the smeltery

### ISSUE THREAD

**Issue by AlmightyAnubis** [ldtteam/minecolonies-features#673]
### Prerequisites 

- [ ] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [x] I made sure that **this change request is not a duplicate of any existing change request.**

### Context 

- Minecraft Version: 1.18.1
- MineColonies Version: minecolonies-1.18.1-1.0.635-BETA.jar
- Related Mods and their Versions:structurize-1.18.1-1.0.378-BETA.jar

## Is your feature request related to a problem?

Before the minecraft Update using fortune on ores like iron and gold the smeltery was very useful because it was the only way to multiply the ores the miner provides you. This already was useful at low levels and was a nice help at the midgame of your colony. But since the update the smeltery is only better than the player using fortune if it is level 5. So the most time, the only use to build the smeltery, is for the research. This is quite sad, because the smeltery got very downgraded by this.

## Enhancement or Change description

Maybe shifting the fortune table so fortune 3 ist at level 3 or implementing the old multiplication table for the raw ores (iron, gold, copper). This will bring the smeltery back to its old glory.
I know, that this has the potential, to generate quite large amount of ingots from low amount of ores, but the amount of ores the miner produces is not so large, especially since you also need miners for your quarry.
So if you want to supply a decent army at midgame you have to build a vanilla iron farm , because this gives so much more iron compared to the iron the colony can produce (they are ridicules and should not be the standard to aim at, I know). If you dont have this much iron, your guards are just there to die to mobs and raiders, as far as I could experience it.

### Notes or related things

---
## Viewers

* Add a thumbs-up to the change request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Talyda** [comment]
Yeah, I'm all for this. The new system just averages out lower than the old one did, and it makes it less useful.

**Comment by AlmightyAnubis** [comment]
@Talyda How would you improve it? I suggested two ways or would you change it in an other way?
